Sources/KSCrashBacktrace/KSBacktrace.m

```diff
@@ -0,0 +1,64 @@
+//
+//  KSBacktrace.m
+//
+//  Created by Alexander Cohen on 2025-05-27.
+//
+//  Copyright (c) 2025 Alexander Cohen. All rights reserved.
```

Comment:
I think we should preserve Karl’s copyright notice in every file.

Reply:
Sure no problem, wasn't sure to do there.